pyaedt/application/Analysis3DLayout.py

```diff
@@ -276,19 +337,20 @@ def create_setup(self, setupname="MySetupAuto", setuptype=None, props={}):
 
     @aedt_exception_handler
     def get_setup(self, setupname, setuptype=None):
-        """Get an existing Setup.
+        """Get an existing setup.
 
         Parameters
         ----------
         setupname : str
-            name of the setup
-        setuptype : SetupTypes
-            optional, setup type. if None, default type will be applied
+            Name of the setup.
+        setuptype : SetupTypes, optional
+            Type of the setup. The default is ``None``, in which case
+            the default type is applied.
 
         Returns
         -------
-        :class: Setup3DLayout
```

Comment:
We can go over this later, but sphinx actually supports these kinds of references.  In the second pass, we can go through and add sphinx directives that allow linking between method docstrings and class documentation.